bids-validator/utils/issues/list.js

```diff
@@ -1108,4 +1108,15 @@ export default {
     severity: 'error',
     reason: 'No valid JSON file found for this file',
   },
+  226: {
+    key: 'UNSUPPORTED_BIG_TIFF',
+    severity: 'error',
+    reason:
+      'BigTiff OME-TIFF file is not supported for consistency check right now, please use Tiff OME-TIFF file instead',
```

Comment:
This should be a warning only.
Big Tiff are supported in the microscopy spec, but not supported in the validator. (BigTiff allows for files over 4 Gb which happens fairly often with microscopy).
So instead of an error, we should just warn the user that the file is a BigTiff, therefore the metadata consistency check is skipped. Ex: `Metadata consistency check skipped for BigTiff OME-TIFF file`.

Reply:
done